File: ts/getting-latest-articles/GetAll.ts
// DocSection: latest_articles_get_all
import { ContentItem, DeliveryClient } from 'kentico-cloud-delivery';

const deliveryClient = new DeliveryClient({
    projectId: '975bf280-fd91-488c-994c-2f04416e5ee3'
});

deliveryClient.items<ContentItem>()
    .getObservable()
    .subscribe(response => console.log(response));
// EndDocSection

File: ruby/getting-latest-articles/GetAll.rb
# DocSection: getting_latest_articles_get_all
require 'delivery-sdk-ruby'

delivery_client = KenticoCloud::Delivery::DeliveryClient.new project_id: '975bf280-fd91-488c-994c-2f04416e5ee3'
delivery_client.items.execute do |response|
  items = response.items
  items.each { |item| puts item.system.codename }
end
# EndDocSection